templates/components/common/ssl-seal.html

```diff
@@ -0,0 +1,8 @@
+<table width="135" border="0" cellpadding="2" cellspacing="0" title="Click to Verify - This site chose GeoTrust SSL for secure e-commerce and confidential communications.">
+    <tr>
+        <td width="135" align="center" valign="top">
+            <script type="text/javascript" src="https://seal.geotrust.com/getgeotrustsslseal?host_name={{theme_settings.geotrust_ssl_seal_hostname}}&amp;size={{theme_settings.geotrust_ssl_seal_size}}&amp;lang={{locale_name}}"></script><br />
+            <a href="http://www.geotrust.com/ssl/" target="_blank"  style="color:#000000; text-decoration:none; font:bold 7px verdana,sans-serif; letter-spacing:.5px; text-align:center; margin:0px; padding:0px;"></a>
+        </td>
+    </tr>
+</table>
```

Comment:
hmmm...  yes, i see. the linter doesn't look at html or json.  wondering if we should add something to do the check.